configs/examples/scale_factors_config.py

```diff
@@ -29,27 +29,27 @@
     "path": "../tea/jsonPOG/POG/MUO/2018_UL/muon_Z.json.gz",
     "type": "NUM_TrackerMuons_DEN_genTracks",
     "year": "2018_UL",
-    "ValType": "sf",
+    "ValType": "nominal",
```

Comment:
I'm wondering why this worked for me with "sf" when I tested yesterday... Let's merge your changes, and then I test again.